llarp/net/route.cpp

```diff
@@ -265,6 +284,24 @@ namespace llarp::net
 #undef MALLOC
 #undef FREE
   }
+
+  std::optional<int>
+  GetIfIndex(std::string ip)
+  {
+    llarp::huint32_t h{};
+    h.FromString(ip);
+    std::optional<int> ret = std::nullopt;
+    ForEachWIN32Interface([&ret, n = ToNet(h)](auto* iface) {
+      if (ret.has_value())
+        return;
+      if (iface->dwForwardNextHop == n.n)
+      {
+        ret = iface->dwForwardIfIndex;
```

Comment:
As in `net/net.hpp`, the `If` here reads like a conditional rather than `interface`